### ISSUE TITLE
tracker binded to wrong foot

### ISSUE THREAD

**Issue by LosWheatleynDew** [nicoco007/BeatSaberCustomAvatars#126]
[2022.01.10.22.56.23.log](https://github.com/nicoco007/BeatSaberCustomAvatars/files/7844350/2022.01.10.22.56.23.log)

Beatsaber 1.18.3
PCVR oculus quest 2
Using april tag fbt
![20220110230039_1](https://user-images.githubusercontent.com/75549894/148896158-eb856423-a4df-4ccb-a9ee-9abbef6d7cee.jpg)
Issue: my trackers on the foot binds to the wrong foot and crosses over instead of the correct foot

**Comment by nicoco007** [comment]
Since the trackers are being detected I'm guessing this is just a tracker assignment issue. Could you try reversing your left and right foot tracker assignments in the SteamVR tracker settings (SteamVR > Settings > Controllers > Manage Vive Trackers)?

**Comment by LosWheatleynDew** [comment]
> Since the trackers are being detected I'm guessing this is just a tracker assignment issue. Could you try reversing your left and right foot tracker assignments in the SteamVR tracker settings (SteamVR > Settings > Controllers > Manage Vive Trackers)?

Yeah i saw the settings for that though i can't switch it. Though its SteamVR or the AprilTagFBT software issue that i'm using and NOT this issue. The simplest solution is to mark what is Left and Right on my trackers and Switch it incase its crossed: This solved the issue

I'm going to close this issue

**Comment by nicoco007** [comment]
Hmm okay, that's strange. Unfortunately I don't have experience with April Tag so I don't know either, but I'm glad you found a workaround. I'm working on a better way to deal with full body trackers that won't require setup through SteamVR so hopefully it'll help in situations like this.
